src/components/backend-ai-session-launcher.ts

```diff
@@ -568,10 +571,6 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
           --mdc-select-min-width: 190px;
         }
 
-        mwc-select > mwc-list-item {
-          width: 370px; // default width
```

Comment:
This style definition enabled horizontal scrollbars, which was not a desirable result. It could be solved by two custom style variables in `mwc-select` element.

When the value of the attribute "fixedMenuPosition" in `mwc-select` set true,  the width of corresponding list-item elements can be set in both `--mdc-menu-min-width` and `--mdc-menu-max-width`. And the value of each custom style variable need to be same as the width of the dialog component. We shall not defining them in `mwc-list-item` directly.We shall not defining them in `mwc-list-item` directly.